xcp_d/workflows/anatomical/surface.py

```diff
@@ -995,35 +1033,36 @@ def init_warp_one_hemisphere_wf(
     # Apply FLIRT-format anatomical-to-template affine transform to 32k surfs
     # I think this makes it so you can overlay the pial and white matter surfaces on the
     # associated volumetric template (e.g., for XCP-D's brainsprite).
-    apply_affine_to_fsLR32k = pe.MapNode(
+    # TODO: Need to loop over affine-warp pairs here rather than apply them in parallel.
+    apply_affine_to_template = pe.MapNode(

```

Comment:
This is the tough section. We have two separate nodes: apply_affine and apply_warpfield, which make sense when there's one of each. However, in the scenario where we have _two_ of each, I'm not sure how to loop over the affine-warpfield pairs and apply these steps twice.

One workaround (which I hate) would be to infer the number of affine-warp pairs in the composite transform outside of the nipype workflow (i.e., using regular Python code in the workflow function) and use that to make a loop using `niu.Select`.